tests/unit/AnnualReport.spec.ts

```diff
@@ -858,8 +858,8 @@ describe('Annual Report - Part 3 - Submitting', () => {
   it('saves a new filing and redirects to Pay URL when this is a new AR and the File & Pay button ' +
     'is clicked', async () => {
     // set necessary session variables
-    sessionStorage.setItem('BASE_URL', `${process.env.VUE_APP_PATH}/`)
-    sessionStorage.setItem('AUTH_WEB_URL', 'auth/')
+    sessionStorage.setItem('BASE_URL', 'https://base.url/')
+    sessionStorage.setItem('AUTH_WEB_URL', 'https://auth.web.url/')
```

Comment:
I changed this to "real" URLs so that `new URL(xxx)` works (since it expects a valid URL), and then didn't end up using that, but I think this is still better code so I left it.